app/code/Magento/PageCache/Test/Mftf/Test/CachedInputFormKeyValueUpdatedTest.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="CachedInputFormKeyValueUpdatedTest">
+        <annotations>
+            <features value="PageCache"/>
+            <stories value="FormKey"/>
+            <title value="Form Key value should be updated by js script"/>
+            <description value="Form Key value should be updated by js script"/>
+            <testCaseId value="MC-39300"/>
```

Comment:
MC-39300 should be in **Ready for review** status

Reply:
updated status